saleor/graphql/app/enums.py

```diff
@@ -20,6 +20,21 @@ def description(enum):
     return None
 
 
+class CircuitBreakerState:
```

Comment:
Would be good to provide a description of each value. Right now, for someone who is not aware of circuit breaker pattern, this will require some googling. We can add some small descriptions to give more input here. Same as done above in line 6.